src/wp-admin/network/site-users.php

```diff
@@ -339,15 +339,15 @@
 
 <?php
 /**
- * Filters whether to show the Add New User form on the Multisite Users screen.
+ * Filters whether to show the Add User form on the Multisite Users screen.
  *
  * @since 3.1.0
  *
- * @param bool $bool Whether to show the Add New User form. Default true.
+ * @param bool $bool Whether to show the Add User form. Default true.
  */
 if ( current_user_can( 'create_users' ) && apply_filters( 'show_network_site_users_add_new_form', true ) ) :
 	?>
-<h2 id="add-new-user"><?php _e( 'Add New User' ); ?></h2>
+<h2 id="add-new-user"><?php _e( 'Add User' ); ?></h2>
```

Comment:
I don't think this can be changed since the above section is "Add Existing User".

Reply:
And also add a comment to tell future us why this one is different.